### PR TITLE
Correct FindBugs-JSR305 license

### PR DESCRIPTION
**Pull request by syleeeee**
I found JSR305 is actually BSD-licensed, not Apache 2.0.
Somehow there's wrong information at maven. 

You can see the discussion regarding the JSR305 license [here](https://github.com/findbugsproject/findbugs/issues/128).

### CODE REVIEW

**Review by ikhoon — APPROVED** [review]
Thanks! @syleeeee 
I didn't know that the jar deployed to Maven Central has wrong information.

**Review by minwoox — APPROVED** [review]
👍 👍 👍

File: NOTICE.txt [hunk 1]
```diff
@@ -174,7 +174,7 @@ This product depends on JCTools, distributed by Nitsan Wakart and contributors:
 
 This product depends on FindBugs-JSR305, distributed by
 
-  * License: licenses/LICENSE.jsr305.al20.txt (Apache License v2.0)
+  * License: licenses/LICENSE.jsr305.bsd.txt (Apache License v2.0)
```
Comment (anuraaga) [inline NOTICE.txt:177]:
```suggestion
  * License: licenses/LICENSE.jsr305.bsd.txt (New BSD License)
```

I think?

**Review by trustin — APPROVED** [review]
Thanks, @syleeeee and @anuraaga !

**Pull request merged**